package.json

```diff
@@ -57,40 +57,40 @@
     "schema-utils": "^4.2.0"
   },
   "devDependencies": {
-    "@babel/cli": "^7.23.9",
-    "@babel/core": "^7.24.0",
-    "@babel/preset-env": "^7.24.0",
-    "@commitlint/cli": "^18.6.1",
-    "@commitlint/config-conventional": "^18.6.2",
+    "@babel/cli": "^7.24.1",
+    "@babel/core": "^7.24.4",
+    "@babel/preset-env": "^7.24.4",
+    "@commitlint/cli": "^19.3.0",
+    "@commitlint/config-conventional": "^19.2.2",
     "@types/file-entry-cache": "^5.0.4",
     "@types/fs-extra": "^11.0.4",
-    "@types/micromatch": "^4.0.6",
-    "@types/node": "^20.11.25",
+    "@types/micromatch": "^4.0.7",
+    "@types/node": "^20.12.7",
     "@types/normalize-path": "^3.0.2",
     "@types/webpack": "^5.28.5",
     "@webpack-contrib/eslint-config-webpack": "^3.0.0",
     "babel-eslint": "^10.1.0",
     "babel-jest": "^29.7.0",
     "chokidar": "^3.6.0",
     "cross-env": "^7.0.3",
-    "cspell": "^8.6.0",
+    "cspell": "^8.7.0",
     "del": "^7.1.0",
     "del-cli": "^5.1.0",
     "eslint": "^8.57.0",
     "eslint-config-prettier": "^9.1.0",
     "eslint-plugin-import": "^2.29.1",
     "file-loader": "^6.2.0",
     "fs-extra": "^11.2.0",
-    "husky": "^8.0.3",
+    "husky": "^9.0.11",
     "jest": "^29.7.0",
     "lint-staged": "^15.2.2",
     "npm-run-all": "^4.1.5",
     "postcss-scss": "^4.0.9",
     "prettier": "^3.2.5",
     "standard-version": "^9.5.0",
-    "stylelint": "^16.2.1",
-    "typescript": "^5.4.2",
-    "webpack": "^5.90.3"
+    "stylelint": "~16.3.1",
```

Comment:
Why it was changed?

Reply:
Version 16.4.0 has typing problems

Reply:
https://github.com/stylelint/stylelint/blob/main/types/stylelint/index.d.ts#L132

https://github.com/cosmiconfig/cosmiconfig/blob/main/src/types.ts#L8C4-L14C10

Reply:
The new version fixes the problem https://github.com/stylelint/stylelint/releases/tag/16.5.0